Library/Formula/subversion.rb

```diff
@@ -156,6 +151,8 @@ def install
     if MacOS::CLT.installed?
       args << "--with-apr=/usr"
       args << "--with-apr-util=/usr"
+      # Ignore authz API bug in system httpd (fixed in httpd 2.4.16)
+      args << "--enable-broken-httpd-auth" if !MacOS::CLT.installed? && MacOS.version < :el_capitan
```

Comment:
`!MacOS::CLT.installed?` will never be fulfilled inside a `if MacOS::CLT.installed?` block. I think the previous version of this line was fine - My question earlier was whether or not we needed to apply the fix to the `Formula["apr"]` usage as well in the block below, rather than just when the system `apr` was used. Apologies if that was unclear.